maistrukia/first/task1.py

```diff
@@ -0,0 +1,9 @@
+"""
+Підрахувати кількість порожніх листів
+"""
+rand_list = ["asdasd", 1231, 1251.1001, [], [123, 123, "ASD"], "KSAJFKJO1", "1234124", [123, 321], [], [], 123, 123.123, [], [123]]
+counter = 0
```

Comment:
`def empty_lists(list_1):
    counter = 0
    for element in list_1:
        if isinstance(element, list) is True and len(element) == 0:
            counter += 1
    return int(counter)
print("Count of empty lists: ", empty_lists(rand_list))`